packages/ChangesReloaded-Tests.package/CRLinter.class/instance/assertAll.areBelowThreshold..st

```diff
@@ -0,0 +1,4 @@
+testing-utils
+assertAll: aCollectionOfResultValues areBelowThreshold: aNumber
+
+	self assert: (aCollectionOfResultValues allSatisfy: [:resultValue | resultValue < aNumber])
```

Comment:
Is `ResultValue` really the type of the variable? Otherwise, `aCollectionOfNumbers` or similar might be reasonable?

Reply:
`allSatisfy:` calls the block for all elements of `aCollectionOfNumbers`, so we thought it would be clear from context that this is a number. So, the role as the parameter name would be more helpful